### ISSUE TITLE
Nudity/Streamer Mode Options Toggle

### ISSUE THREAD

**Issue by SlamBamActionman** [space-wizards/space-station-14#31632]
## Description

It's desirable to have an option to toggle nudity on/off, which covers up all nudity using simple gender-neutral clothing on the user's client. This is due to accessibility concerns (even if low-resolution pixelated, nudity may cause uncomfortableness) and to make the game more streamer-friendly. 

This feature is requested alongside the implementation of #31221; this is because that PR implements a system that works per-character and is synced across the server, while the feature requested here is client-side only and applies on all characters.

The nudity toggle should be an option in the Accessibility options tab. This toggle should respect a character's specific choice of underwear if one exists, but if a character is lacking either a bottom or top undergarment then it should default to boxer briefs and tanktop respectively. There should not be any distinction made between genders or bodytypes.

### Desired behavior
When the toggle is off, User A sees User B's character on their client as either being naked or wearing underwear, depending on the undergarments set in User B's character creation.

When the toggle is on, User A sees User B's character on their client with underwear on, always. If User B has set underwear during character creation then that is used, but if not it defaults to boxer briefs/tanktop.

**Comment by Dutch-VanDerLinde** [comment]
btw in the underwear pr you cannot take off bottom underwear

**Comment by ArtisticRoomba** [comment]
Merging PR #33185 will close this issue

**Comment by ScarKy0** [comment]
Resolved by #33185
